src/components/common/FilterMultiSelectContainer/index.tsx

```diff
@@ -0,0 +1,121 @@
+import React, { useState } from 'react';
+import { Popover, Theme, Typography } from '@mui/material';
+import { makeStyles } from '@mui/styles';
+import Icon from 'src/components/common/icon/Icon';
+import useDeviceInfo from 'src/utils/useDeviceInfo';
+import FilterMultiSelect from 'src/components/common/FilterMultiSelect';
+
+const useStyles = makeStyles((theme: Theme) => ({
+  dropdown: {
+    cursor: 'pointer',
+    border: `1px solid ${theme.palette.TwClrBrdrSecondary}`,
+    borderRadius: '4px',
+    width: '176px',
+    height: '40px',
+    padding: theme.spacing(1, 2, 1, 1),
+    marginTop: theme.spacing(0.5),
+    display: 'flex',
+    justifyContent: 'space-between',
+  },
+  dropdownDisabled: {
+    color: theme.palette.TwClrTxtTertiary,
+  },
+  dropdownIconRight: {
+    height: '24px',
+    width: '24px',
+  },
+  popoverContainer: {
+    '& .MuiPaper-root': {
+      borderRadius: '8px',
+      overflow: 'visible',
+      width: '480px',
+    },
+  },
+  mobileContainer: {
+    borderRadius: '8px',
+    overflow: 'visible',
+    position: 'fixed',
+    top: '50%',
+    left: '50%',
+    transform: 'translate(-50%, -50%)',
+    maxHeight: '90%',
+    width: '90%',
+    zIndex: 1300,
+  },
+}));
+
+type FilterMultiSelectContainerProps<T> = {
+  filters: T;
+  setFilters: (f: T) => void;
+  label: string;
+  disabled?: boolean;
+  filterKey: keyof T;
+  options: number[];
+  renderOption: (id: number) => string;
+};
+
+export default function FilterMultiSelectContainer<T extends Record<string, number[]>>(
```

Comment:
Just thinking of whether might need to be `<T extends Record<string, (number | string)[]>>` but honestly, if that's needed we can always add it later